### PR TITLE
add intro

### PR DESCRIPTION
**Pull request by sophiadicuffa**

### CODE REVIEW

File: LabOne.tex [hunk 1]
```diff
@@ -0,0 +1,21 @@
+\chapter{Bookstore UML Modeling \\
+  \small{\textit{-- Evan Ciok, Sophia DiCuffa, Carson McManus}}
+  \index{Chapter!introduction}
```
Comment (dyc3) [inline LabOne.tex]:
```suggestion
  \index{Chapter!labone}
```

File: LabOne.tex [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    \scalebox{0.8}{\includegraphics{Figures/bookstore/usecase.jpg}}
+    \caption{\label{Figure::useCase} A use-case diagram for the e-Bookstore system that supports the list of functions shown above.}
+    \end{figure}
+
+    \begin{figure}
+        \centering
+        \scalebox{0.8}{\includegraphics{Figures/bookstore/uml.png}}
+        \caption{\label{Figure::useCase} A class diagram for the e-Bookstore system that illustrates the choices of the main classes and 
+        appropriate relationships among them.}
+        \end{figure}
```
Comment (dyc3) [inline LabOne.tex]: nit: atrocious whitespace formatting lol

File: manual.tex [hunk 3]
```diff
@@ -281,7 +282,7 @@
 
 \singlespacing
 \include{manIntroduction}
-
+\include{LabOne.tex}
```
Comment (dyc3) [inline manual.tex]: file name should be `manLabOne.tex` as specified in the assignment

File: LabOne.tex [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+\begin{figure}
+    \centering
+    \scalebox{0.8}{\includegraphics{Figures/bookstore/usecase.jpg}}
+    \caption{\label{Figure::useCase} A use-case diagram for the e-Bookstore system that supports the list of functions shown above.}
+    \end{figure}
+
+    \begin{figure}
+        \centering
+        \scalebox{0.8}{\includegraphics{Figures/bookstore/uml.png}}
+        \caption{\label{Figure::useCase} A class diagram for the e-Bookstore system that illustrates the choices of the main classes and
```
Comment (dyc3) [inline LabOne.tex]:
```suggestion
        \caption{\label{Figure::bookstoreClass} A class diagram for the e-Bookstore system that illustrates the choices of the main classes and 
```

File: LabOne.tex [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  \index{Lab One}
+  \label{Chapter::LabOne}}
+
+% Add a section and label it so that we can reference it later
+\section{Bookstore UML Modeling \label{Section::LabOne}}
+
+\begin{figure}
+    \centering
+    \scalebox{0.8}{\includegraphics{Figures/bookstore/usecase.jpg}}
+    \caption{\label{Figure::useCase} A use-case diagram for the e-Bookstore system that supports the list of functions shown above.}
```
Comment (dyc3) [inline LabOne.tex]:
```suggestion
    \caption{\label{Figure::bookstoreUseCase} A use-case diagram for the e-Bookstore system that supports the list of functions shown above.}
```

**Review by dyc3 — CHANGES_REQUESTED** [review]

File: LabOne.tex (hunk 2 above)
Comment (sophiadicuffa) [reply LabOne.tex]: i cant see it on my end so idk how to fix :/ i just copied the prof's example

Comment (dyc3) [reply LabOne.tex]: I was talking about the litteral whitespace indentation in the latex code, not how it renders in the pdf.

Comment (sophiadicuffa) [reply LabOne.tex]: ohh my b

**Review by dyc3 — APPROVED** [review]

**Comment by dyc3** [comment]
Currently struggling to get it to render but I'll fix it

**Pull request merged**